packages/framework/src/Form/ImageUploadType.php

```diff
@@ -118,7 +138,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             'multiple' => $this->isMultiple($options),
             'mapped' => false,
             'attr' => [
-                'accept' => 'image/*',
+                'accept' => 'image/gif|image/jpeg|image/png',
```

Comment:
I would like to have a definition of supported types in one place. I can imagine the WTF moment when I add an image type and can't select it in the browser :smile:

Reply:
changed

Reply:
Nice, thank you!